src/libsystemd/sd-bus/bus-internal.c

```
@@ -91,8 +92,17 @@ bool interface_name_is_valid(const char *p) {
                                 (!dot && *q >= '0' && *q <= '9') ||
                                 *q == '_';
 
-                        if (!good)
+                        if (!good) {
+                                if (DEBUG_LOGGING) {
+                                        _cleanup_free_ char *iface = NULL;
+                                        iface = cescape(p);
+                                        if (!iface)
+                                                log_oom();
```

We generally don't suppress the error from oom in such scenarios:
```C
if (DEBUG_LOGGING) {
     _cleanup_free_ char *iface = cescape(p);
     log_debug(..., strnull(iface), ...);
```